nvtabular/ops/categorify.py

```diff
@@ -30,6 +30,7 @@
 from dask.delayed import Delayed
 from dask.highlevelgraph import HighLevelGraph
 from fsspec.core import get_fs_token_paths
+from nvtx import annotate
```

Comment:
We shouldn't be importing nvtx.annotate directly anymore - we should be using the version inside nvtabular.dispatch (contains a fallback for case where nvtx isn't installed, like CPU only systems)

Reply:
yeah this is a draft still.